### PR TITLE
reg tests and docs

### LINKED ISSUE #372: compute kinetic energy/enstrophy
move these methods out of incflo.H and into a PostProcessing /DerivedQty type object

### PR DESCRIPTION
**Pull request by michaeljbrazell**
- removed old input for kinetic energy
- updated reg tests
- removed pressure output from vof reg tests
- updated docs
Fixes #372

### CODE REVIEW

**Review by jrood-nrel — APPROVED** [review]
I just looked through all the changes and it looks straightforward to me.

**Review by sayerhs — APPROVED** [review]

File: docs/sphinx/user/inputs.rst [hunk 1]
```diff
@@ -1166,3 +1168,26 @@ Example::
...
+Total Enstrophy
+`````````````````
+
+The ``Enstrophy`` sampling integrates the enstrophy over the domain and
+outputs it to a text file identified using the label and output in 
+the post_processing directory.
+
+Example::
+
+  sampling.enst1.type              = Enstrophy
```
Comment (sayerhs) [inline docs/sphinx/user/inputs.rst:1192]: Why is there a sampling prefix here? Should this be the keyword provided in postprocessing section?

File: test/test_files/ctv_godunov_plm/ctv_godunov_plm.i [hunk 2]
```diff
@@ -15,7 +15,6 @@ time.cfl              =   0.5        # CFL factor
 #.......................................#
 time.plot_interval  =  20   # Steps between plot files
 time.checkpoint_interval =   -1  # Steps between checkpoint files
-io.KE_int = 1        # calculate kinetic energy
```
Comment (sayerhs) [inline test/test_files/ctv_godunov_plm/ctv_godunov_plm.i:18]: Is there a regression text that exercises ke and enstrophy computation?

File: docs/sphinx/user/inputs.rst (hunk 1 above)
Comment (michaeljbrazell) [reply docs/sphinx/user/inputs.rst:1192]: oh good catch, I'll fix it

File: test/test_files/ctv_godunov_plm/ctv_godunov_plm.i (hunk 2 above)
Comment (michaeljbrazell) [reply test/test_files/ctv_godunov_plm/ctv_godunov_plm.i:18]: yep, tgv_godunov does

**Pull request merged**